radiography/src/main/java/radiography/Radiography.kt

```diff
@@ -76,15 +76,18 @@ object Radiography {
 
     for (view in matchingRootViews) {
       if (length > 0) {
+        @Suppress("DEPRECATION")
         appendln()
       }
       val layoutParams = view.layoutParams
       val title = (layoutParams as? WindowManager.LayoutParams)?.title?.toString()
           ?: view.javaClass.name
+      @Suppress("DEPRECATION")
       appendln("$title:")
 
       val startPosition = length
       try {
+        @Suppress("DEPRECATION")
```

Comment:
What's up with those? Should we use something else?

Reply:
Yes, but we can't we're not building with 1.3 anymore. They're deprecated in 1.4.